app/src/main/java/at/bitfire/icsdroid/ui/InfoActivity.kt

```diff
@@ -72,4 +84,118 @@ class InfoActivity: AppCompatActivity() {
         }
     }
 
+    class LibsFragment: Fragment() {
+        @Composable
+        fun TextDialog(@StringRes descriptionRes: Int, onDismissRequest: () -> Unit) {
+            AlertDialog(
+                onDismissRequest = onDismissRequest,
+                confirmButton = {
+                    TextButton(onClick = onDismissRequest) {
+                        Text(stringResource(android.R.string.ok))
+                    }
+                },
+                text = {
+                    Text(
+                        text = stringResource(descriptionRes),
+                        modifier = Modifier.fillMaxWidth()
+                    )
+                }
+            )
+        }
+
+        @Composable
+        fun InfoHeader() {
+            val context = LocalContext.current
+
+            var showingLicenseDialog by remember { mutableStateOf(false) }
+            if (showingLicenseDialog)
+                TextDialog(R.string.app_info_gplv3_note) { showingLicenseDialog = false }
+
+            var showingDonateDialog by remember { mutableStateOf(false) }
+            if (showingDonateDialog)
+                TextDialog(R.string.donate_message) { showingDonateDialog = false }
+
+            Column(modifier = Modifier.fillMaxWidth()) {
```

Comment:
One more remark: 
If you put here in the column 
`horizontalAlignment = Alignment.CenterHorizontally,`
you can probably delete all the lines with
`modifier = Modifier.fillMaxWidth()`
and
`textAlign = TextAlign.Center,`
😉